### ISSUE TITLE
run_example.sh with pretrained models give only one speaker label on random .wav clips

### ISSUE THREAD

**Issue by shamilajeewantha** [BUTSpeechFIT/DiaPer#9]
I tried using run_example.sh script on some random wav files other than IS1009a.wav and it results in the same speaker label spk0 for all the time segments. Are there any specific conversion methods I should follow? 

The wav file details with `soxi `command are as follows and I used ffmpeg for mp3 to wav conversion.

```
Input File     : 'IS1009a.wav'
Channels       : 1
Sample Rate    : 16000
Precision      : 16-bit
Duration       : 00:13:58.83 = 13421333 samples ~ 62912.5 CDDA sectors
File Size      : 26.8M
Bit Rate       : 256k
Sample Encoding: 16-bit Signed Integer PCM


Input File     : 'CAR0001.wav'
Channels       : 1
Sample Rate    : 16000
Precision      : 16-bit
Duration       : 00:10:25.77 = 10012302 samples ~ 46932.7 CDDA sectors
File Size      : 20.0M
Bit Rate       : 256k
Sample Encoding: 16-bit Signed Integer PCM
```

the wav file I tested and the rttm results are attached below
[CAR0001.zip](https://github.com/user-attachments/files/17409028/CAR0001.zip)

**Comment by fnlandini** [comment]
Hi, after listening to the file, I would say that it should be a relatively easy situation. The model used in the example script was only trained on simulated data and that could be a problem. I suggest you try replacing in run_example.sh `--models-path $CDIR/models/10attractors/SC_LibriSpeech_2spk_adapted1-10/models/` by `--models-path $CDIR/models/20attractors/SC_LibriSpeech_2spk_adapted1-10_finetuneRAMC/models` so that it uses a different model. This one was fine-tuned to a set with telephone conversations between two speakers so it should be a better match.
This different model also has a slightly different configuration so in [this line](https://github.com/BUTSpeechFIT/DiaPer/blob/3c4af3cbc8e9c1840a276cc07f2c564adcd6bb2e/examples/infer_16k_10attractors.yaml#L23) you will have to replace `10` by `20` and the [epochs](https://github.com/BUTSpeechFIT/DiaPer/blob/3c4af3cbc8e9c1840a276cc07f2c564adcd6bb2e/examples/infer_16k_10attractors.yaml#L7) from `90-100` to `366-375`
You could also try some of the other models in https://github.com/BUTSpeechFIT/DiaPer/tree/main/models and I recommend you use the ones that are already finetuned.

I hope this helps
Federico

**Comment by shamilajeewantha** [comment]
Thank you a lot for the quick reply. I tried the other models as you suggested and it worked a lot better than the initial one.
